README.md

```diff
@@ -241,63 +241,6 @@ $ git reset -- hard commit cf5d4d0d3808cef721f84a812da8e99641ce5666
 # bundle 02
 # exercise 01
 ```bash
-User@DESKTOP-Q9JB4D6 MINGW64 ~/Desktop/The Gym/Gym-Git-Exercise-Solutions1 (main)
```

Comment:
Add Your Terminal History to the README File.
And remember to separate the exercises and bundles.